debian/control

```diff
@@ -0,0 +1,16 @@
+Source: sio2jail
+Section: utils
+Priority: optional
+Maintainer: Michał Sidor <michcioperz@asirpa.iscute.ovh>
```

Comment:
nit: that's not very up to date lol

Reply:
Yep, I will change maintainer to sio2project